weblate/memory/machine.py

```diff
@@ -72,4 +74,5 @@ def download_translations(
                 "origin": result.get_origin_display(),
                 "source": result.source,
                 "show_quality": True,
+                "delete_url": reverse("api:memory-detail", kwargs={"pk": result.id}),
```

Comment:
The delete URL should be included only in case user has a permission to delete. Otherwise the UI shows button which does not work. Check for `user.has_per("memory.delete", result)` is thus needed here.